src/phockup.py

```diff
@@ -344,32 +357,40 @@ def get_file_name_and_path(self, filename):
         target_file_path = os.path.sep.join([output, target_file_name])
         return output, target_file_name, target_file_path, target_file_type
 
-    def process_xmp(self, original_filename, file_name, suffix, output):
+    def process_sidecars(self, original_filename, file_name, suffix, output):
         """
-        Process xmp files. These are metadata for RAW images
+        Given an existing image, handle any sidecar files.
         """
-        xmp_original_with_ext = original_filename + '.xmp'
-        xmp_original_without_ext = os.path.splitext(original_filename)[0] + '.xmp'
-
+        car_no_ext, car_extension = os.path.splitext(original_filename)
```

Comment:
Can we change `car_*` to `sc_*` or `scar_*`.

Reply:
Actually I have to retract that comment, these variables are named appropriately. These variables have to do with the primary image file - if you call accessory files "sidecars", that makes the main image the "car". It doesn't make sense to rename these `sc` or `scar` because they aren't sidecar related.